app/controllers/api/v1/admin/communications_controller.rb

```diff
@@ -34,6 +35,19 @@ def destroy
 
         private
 
+        def update_params
+          params.require(:communication).permit(:title, :text, :published, :recurrent_on)
```

Comment:
Just to be sure, you can't make a communication recurrent once created?

Reply:
my bad